cypress/e2e/person.cy.js

```diff
@@ -179,11 +179,11 @@ describe('Person page', {'tags': tags}, ()=> {
             type: tenureModel.tenureType.description,
         };
 
-        addTestRecordToDatabase("Assets", assetModel);

```

Comment:
If we have an id at this point in time, won't it be within the given entity model (2nd arg) already?

Reply:
Yes, we should be able to use the `seedDatabase` methods we already have set up. 

Tbh, this code isn't related to the PR, I was just updating all of the references to the method I changed. But I agree, it needs to be changed.